data/text_sampling/text_sampling.py

```diff
@@ -969,8 +970,8 @@ def apply_sampling_and_export(
     # path_data_dir = path_data_dir[index:]
 
     for path in path_data_dir:
-        if path.find("data/tabular/bicerano_dataset") != -1:
-            continue  # needs the units fixed or templates copied over
+        # if "smiles_to_3d" not in path:
```

Comment:
Will we still need that?

Reply:
this is a debugging comment as all the ones below